pwndbg/aglib/nearpc.py

```diff
@@ -168,23 +173,41 @@ def nearpc(
 
     assembly_strings = D.instructions_and_padding(instructions)
 
+    breakpoint_locations = pwndbg.dbg.breakpoint_locations()
+
+    prefix_sign = pwndbg.config.nearpc_prefix
+    breakpoint_sign = pwndbg.config.nearpc_breakpoint_prefix
+
     # Print out each instruction
     for i, (address_str, symbol, instr, asm) in enumerate(
         zip(addresses, symbols, instructions, assembly_strings)
     ):
-        prefix_sign = pwndbg.config.nearpc_prefix
-
         # Show prefix only on the specified address and don't show it while in repeat-mode
         # or when showing current instruction for the second time
         show_prefix = instr.address == pc and not repeat and i == index_of_pc
-        prefix = " %s" % (prefix_sign if show_prefix else " " * len(prefix_sign))
-        prefix = c.prefix(prefix)
-
+        is_breakpoint = False
+        if show_prefix:
+            prefix = f" {prefix_sign}"
+            prefix = c.prefix(prefix)
+        elif instr.address in breakpoint_locations:
+            # If the instruction is not the current instruction and a breakpoint,
+            # show the breakpoint sign
+            prefix = breakpoint_sign.ljust(len(prefix_sign) + 1)
+            prefix = c.breakpoint(prefix)
+            is_breakpoint = True
+        else:
+            prefix = " " * (len(prefix_sign) + 1)
+            prefix = c.prefix(prefix)
```

Comment:
Another nitpick: we should probably compute `if show_prefix` and `else` outside of the loop and here just do if instr addr not in bps - use default prefix, otherwise use the breakpoint one